utils/generateData.js

```diff
@@ -0,0 +1,57 @@
+const faker = require('faker');
+const fs = require('fs');
+const { Readable } = require('stream');
+const {
+  getLikes,
+  generateDate,
+  randomNum,
+} = require('../server/database/seedingUtils');
+
+const MIN_UPDATES = 2;
+const MAX_UPDATES = 10;
+
+class GenDataStream extends Readable {
+  constructor(opt) {
+    super(opt);
+
+    this.projects = 10000000;
```

Comment:
Perhaps write this in a format of 10 * 1000000 so you'll know how many millions you're working with?

Reply:
@vyoung415 Fixed, and corrected a few errors.